netbout-client/src/main/java/com/netbout/mock/MkAttachment.java

```diff
@@ -117,6 +120,44 @@ public boolean unseen() {
     }
 
     @Override
+    public Date date() throws IOException {
+        try {
+            return new JdbcSession(this.sql.source())
+                .sql("SELECT date FROM attachment WHERE bout = ? AND name = ?")
+                .set(this.bout)
+                .set(this.label)
+                .select(
+                    new Outcome<Date>() {
```

Comment:
Can't use `new SingleOutcome<>(Utc.class)).getDate()` here. This is because dates are always stored in `H2Sql` using local time zone, and then they should be retrieved again with the same local time zone, not UTC (which is the only one available as an `Outcome`).